### PR TITLE
Removes freon

### PR DESCRIPTION
**Pull request by AnturK**
Note to self : don't ask people for things to remove. Damn you ~~@lzimann~~ @ChangelingRain 

SM balancing/change is bad and freezethrower igniters break thermodynamics but this offer didn't guarantee proper replacements.

- Removes freon.
- Adds freezethrowers for SM emergencies/grief.
- Fixes flamethrowers not working on diagonals.

### CODE REVIEW

**Comment by evsey9** [comment]
but why though

**Comment by lzimann** [comment]
Hey to be fair it wasn't me that asked you to remove it, but I support it anyway.

**Comment by evsey9** [comment]
tell us how do freezethrowers work, and how are they simpler than freon

**Comment by XDTM** [comment]
Does this transfer freon's properties to cold gases in general then?

**Comment by AnturK** [comment]
They're still bad, the whole proc needs renaming and externalizing. I just had enough of changing stuff for other people. @ChangelingRain  can finish this.

**Comment by AnturK** [comment]
@XDTM No. Wasn't a pr for this open before. Should be easy to do though.

**Comment by Iamgoofball** [comment]
Can you guys wait for @duncathan on this one he had plans for this and I'm not happy you're removing the gas because "muh OP".

File: code/game/objects/items/weapons/flamethrower.dm [hunk 1]
```diff
@@ -218,30 +225,63 @@
...
+	create_full = TRUE
+
+/obj/item/weapon/flamethrower/freeze/full/tank
+	create_with_tank = TRUE
+
+/obj/item/device/assembly/igniter/cold/ignite_turf(obj/item/weapon/flamethrower/F,turf/open/location,release_amount = 0.05)
+	if(!F.ptank || !istype(location))
+		return
+	var/datum/gas_mixture/air_transfer = F.ptank.air_contents.remove_ratio(release_amount)
+	if(air_transfer.gases["plasma"])
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/flamethrower.dm]: Why is it plasma

Why do you get to bitch about UGH FREEON SO UNREALISTIC

then immediately turn around and make Plasma do it

**Comment by Iamgoofball** [comment]
Oh, and this can't be merged until you recompile tgui.

Suck my dick.

**Comment by Iamgoofball** [comment]
Oh, and this can't be merged until you recompile tgui.

Suck my dick.

**Comment by ChangelingRain** [comment]
recompiling tgui is easy tho

**Comment by Bawhoppen** [comment]
I can understand this PR but I think it'd be better if we fixed it instead of removing it

**Comment by optimumtact** [comment]
I like freon, I think it is an interesting gas that has potential to be really useful if it was cleaned up. So on that basis I oppose this PR :-1: 

I also told Dunc not to merge that other pr

**Comment by duncathan** [comment]
freon is more interesting than this; as much as i hate freon it is still more interesting than a lot of atmos stuff so it's worth keeping unless a noticeably better alternative is presented

**Pull request closed without merging**